backend/packages/Upgrade/src/api/services/SegmentService.ts

```diff
@@ -160,8 +302,20 @@ export class SegmentService {
     });
     const allSegmentsData = await this.getSegmentByIds(allSegmentIds);
     const duplicateSegmentsIds = allSegmentsData?.map((segment) => segment?.id);
+    const contextMetaData = env.initialization.contextMetadata;
+    const contextMetaDataOptions = Object.keys(contextMetaData);
 
     for (const segment of segments) {
```

Comment:
This is not purely importing the segments but also doing validation like context should exist and all the sub-segments should exist before the actual import. These errors should be sent back to the user.

Reply:
we won't throw error if wrong group name is added, we decided to just ignore that row and create that segment although we throw error on wrong subsegment so I think we should have same approach for both. Which one do you suggest?

Reply:
changed